Tests/MarathonTests/MarathonTests.swift

```diff
@@ -298,26 +298,39 @@ class MarathonTests: XCTestCase {
     }
 
     func testInstallingRemoteScriptWithDependencies() throws {
-        try run(with: [
-            "install",
-            "https://raw.githubusercontent.com/JohnSundell/Marathon-Examples/master/AddSuffix/addSuffix.swift",
-            "installed-script"
-        ])
-
-        // Make a couple of files that we can try the installed script on
-        let executionFolder = try folder.createSubfolder(named: "TestInstallation")
-        try executionFolder.createFile(named: "A.swift")
-        try executionFolder.createFile(named: "B.swift")
-
-        // Run the installed binary
-        try executionFolder.moveToAndPerform(command: "../installed-script -suffix")
-        XCTAssertEqual(executionFolder.files.names, ["A-suffix.swift", "B-suffix.swift"])
-
-        // List should not contain the script, as it was only added temporarily
-        try XCTAssertFalse(run(with: ["list"]).lowercased().contains("addsuffix"))
-
-        // Make sure that the temporary folder for the script is cleaned up
-        try XCTAssertEqual(folder.subfolder(atPath: "Scripts/Temp").subfolders.count, 0)
+    
+        let rawGithubUrlString = "https://raw.githubusercontent.com/JohnSundell/Marathon-Examples/master/AddSuffix/addSuffix.swift"
+        
+        let githubUrlString = "https://github.com/JohnSundell/Marathon-Examples/blob/master/AddSuffix/addSuffix.swift"
+        
+        // Check both raw and regular github url types
+        for scriptUrlString in [rawGithubUrlString, githubUrlString] {
+            
+            setUp()
```

Comment:
Yeah I agree... It was lazy of me to solve it that way to avoid code repetition 🙈 Hopefully you'll find current solution more suitable